### PR TITLE
test: ceph-objectstore-tool: should import platform before using it

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

File: src/test/ceph_objectstore_tool.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def check_output(*popenargs, **kwargs):
 import logging
 import json
 import tempfile
-import io
```
Comment (oms4suse) [inline src/test/ceph_objectstore_tool.py]: io is definitely not used in this module so +1

File: src/test/ceph_objectstore_tool.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def check_output(*popenargs, **kwargs):
...
+import platform
```
Comment (oms4suse) [inline src/test/ceph_objectstore_tool.py]: platform is definitely used so another +1

File: src/test/ceph_objectstore_tool.py [hunk 3]
```diff
@@ -1967,7 +1967,7 @@ def main(argv):
 
 def remove_btrfs_subvolumes(path):
     if platform.system() == "FreeBSD":
-        return
```
Comment (oms4suse) [inline src/test/ceph_objectstore_tool.py]: removing trailing whitespace is always good.

**Comment by tchaikov** [comment]
> 80/153 Test   #3: ceph_objectstore_tool.py ................   Passed  262.10 sec

**Pull request merged**